relays/client-substrate/src/chain.rs

```diff
@@ -276,6 +278,21 @@ where
 	}
 }
 
+/// Structure that implements `UtilityPalletProvider` based on a call conversion.
+pub struct MockedRuntimeUtilityPallet<Call> {
```

Comment:
I dont know, I just cannot get used to `Mock` word for production code :D
Wdyt about `WrappedRuntimeUtilityPallet`  (we have runtime_wrapper.rs in clients) or just `RuntimeUtilityPallet` or some other equivalent to `FullRuntimeUtilityPallet`?
Also, maybe in near future we will have `CodegenRuntimeUtilityPallet` for Serban's subxt stuff